utils/kernel-refresh.sh

```diff
@@ -1,26 +1,32 @@
 #!/bin/sh
 
 KDIR=$1
-KTAG=$2
-PDIR=$3
-
+OLD_VER=$2
+KTAG=$3
+PDIR=$4
+DDIR=$5
+NEW_VER=$6
 usage()
 {
     cat <<EOF
-usage: $0 <kernel-dir> [<kernel-tag>] [<patch-dir>]
+usage: $0 <kernel-dir> <old-version> [<kernel-tag>] [<patch-dir>] [defconfig-dir] [new-version]
```

Comment:
These four optional positional arguments look spicy. Do we really support all 16 combinations? Maybe we should enlist `getopts` for some assistance?